src/app/_services/container/container.service.ts

```diff
@@ -3,18 +3,19 @@ import {Observable} from 'rxjs/Observable';
 import {HttpClientModule, HttpClient, HttpParams} from '@angular/common/http';
 import {Container} from '../../_models';
 import {map} from 'rxjs/operators';
+import {ConfigurationService} from '../configuration/configuration.service';
 
 
 
 @Injectable()
 export class ContainerService {
   private host: string;
-  constructor(private http: HttpClient) {
+  constructor(private http: HttpClient, private config: ConfigurationService) {
         this.host = 'http://localhost:8080';
```

Comment:
Is this variable still used?

Reply:
Do I make a single commit to remove that :peanuts: 

I wonder why I didn't get any warnings :/

Reply:
Single commit is fine, a description next to the emoji would help to quickly identify the meaning of the icon :wink:

Reply:
xDDDD Was getting lazy with all the small changes

Thanks @egeldenhuys !